src/PresentationalComponents/ZeroState/AppZeroState.js

```diff
@@ -19,6 +19,7 @@ const AppZeroState = ({ app, customInstructions, customButton, customText, custo
                     customTitle={customTitle}
                     appId={appId}
                 />
+                xd
```

Comment:
Before this gets merged, let's remove this.

Reply:
Nice catch. :D

Reply:
Thanks this one debugging tool got away! :smile: